packages/moonstone/Input/InputSpotlightDecorator.js

```diff
@@ -27,6 +27,24 @@ const handleKeyDown = handle(
 	call('onKeyDown')
 );
 
+
+/**
+ * Default config for {@link moonstone/Input.InputSpotlightDecorator}
```

Comment:
Not `Default config for {@link moonstone/Input.InputSpotlightDecorator}`
but `Default config for [`InputSpotlightDecorator`]{@link moonstone/Input/InputSpotlightDecorator.InputSpotlightDecorator}`

The format is [`config name`]{@link path/moduleName.componentName}.

Reply:
Fixed